build.yaml

```
@@ -1,25 +1,14 @@
 name: "TVHeadend"
 guid: "3fd018e5-5e78-4e58-b280-a0c068febee0"
 imageUrl: "https://repo.jellyfin.org/releases/plugin/images/jellyfin-plugin-tvheadend.png"
-version: 11
-targetAbi: "10.8.0.0"
+version: 12
+targetAbi: .0
```

```suggestion
version: 11
targetAbi: 10.9.0.0
```

also change `framework` to `net8.0`